app/javascript/packs/document-capture.jsx

```diff
@@ -21,20 +22,22 @@ const device = {
     /ip(hone|ad|od)|android/i.test(window.navigator.userAgent),
 };
 
-const appRoot = document.getElementById('document-capture-form');
-const isLivenessEnabled = appRoot.hasAttribute('data-liveness');
-render(
-  <AcuantProvider
-    credentials={getMetaContent('acuant-sdk-initialization-creds')}
-    endpoint={getMetaContent('acuant-sdk-initialization-endpoint')}
-  >
-    <I18nContext.Provider value={i18n.strings}>
-      <AssetContext.Provider value={assets}>
-        <DeviceContext.Provider value={device}>
-          <DocumentCapture isLivenessEnabled={isLivenessEnabled} />
-        </DeviceContext.Provider>
-      </AssetContext.Provider>
-    </I18nContext.Provider>
-  </AcuantProvider>,
-  appRoot,
-);
+loadPolyfills(['fetch']).then(() => {
```

Comment:
Could be simplified in the future, avoiding the callback altogether:

```suggestion
await loadPolyfills(['fetch']);
```

- https://webpack.js.org/configuration/experiments/ (`experiments.topLevelAwait`)
- https://github.com/tc39/proposal-top-level-await